### PR TITLE
Add body on post

### PR DESCRIPTION
**Pull request by antban**
# One-line summary

Return created event type definition in body response on `POST /event-types` under feature toggle

### CODE REVIEW

File: api-metastore/src/main/java/org/zalando/nakadi/controller/EventTypeController.java [hunk 1]
```diff
@@ -90,7 +91,13 @@ public EventTypeController(final EventTypeService eventTypeService,
 
         eventTypeService.create(eventType, true);
 
-        return ResponseEntity.status(HttpStatus.CREATED).headers(generateWarningHeaders(eventType)).build();
+        final ResponseEntity.BodyBuilder bodyBuilder = status(HttpStatus.CREATED)
+                .headers(generateWarningHeaders(eventType));
+        if (featureToggleService.isFeatureEnabled(RETURN_BODY_ON_CREATE_EVENT_TYPE)) {
+            return bodyBuilder.body(eventTypeService.get(eventType.getName()));
```
[the remark this answers is not in the document]
Comment (antban) [reply api-metastore/src/main/java/org/zalando/nakadi/controller/EventTypeController.java:97]: @a1exsh Either `build()` or `body()`

**Comment by antban** [comment]
:+1:

File: docs/_data/nakadi-event-bus-api.yaml [hunk 2]
```diff
@@ -183,6 +183,8 @@ paths:
       responses:
         '201':
           description: Created
+          schema:
+            $ref: '#/definitions/EventType'
```
Comment (a1exsh) [inline docs/_data/nakadi-event-bus-api.yaml:187]: Should add the same for `/event-types/{name} put` then?

Comment (antban) [reply docs/_data/nakadi-event-bus-api.yaml:187]: @a1exsh  Definitely!

**Comment by a1exsh** [comment]
:+1:

**Comment by antban** [comment]
@a1exsh Seems that your approval is not enough...

**Comment by a1exsh** [comment]
:+1:

**Comment by thomasabraham** [comment]
👍

**Comment by antban** [comment]
:+1:

**Pull request merged**